src/lib/reasoners/enum_rel.ml

```diff
@@ -44,11 +46,28 @@ module HSS = Set.Make (struct type t=Hs.t let compare = Hs.compare end)
 
 module LR = Uf.LX
 
-type t = { mx : (HSS.t * Ex.t) MX.t; classes : Expr.Set.t list;
-           size_splits : Numbers.Q.t }
-
-let empty classes = { mx = MX.empty; classes = classes;
-                      size_splits = Numbers.Q.one }
+type t = {
+  mx : (HSS.t * Ex.t) MX.t;
+  (* Map of uninterpreted enum semantic values to domains of their possible
+     values. The explanation justifies that any value assigns to the semantic
+     value has to lie in the domain. *)
```

Comment:
I changed a bit the comment. It didn't make sense to talk about `the value of the semantic value`.